### PR TITLE
Memcache ready for new Prometheus setup

### PR DESCRIPTION
**Pull request by auhlig**

### CODE REVIEW

File: common/memcached/templates/alerts/_memcached.alerts.tpl [hunk 1]
```diff
@@ -0,0 +1,14 @@
+groups:
+- name: memcached.alerts
+  rules:
+  - alert: {{ include "alerts.service" . | title }}MemcachedManyConnectionsThrottled
+    expr: (rate(memcached_connections_yielded_total{app="barbican-memcached"}[5m]) * 60) > 5
```
Comment (reimannf) [inline common/memcached/templates/alerts/_memcached.alerts.tpl]: guess that should be generalised

Comment (auhlig) [reply common/memcached/templates/alerts/_memcached.alerts.tpl]: Indeed. Thanks for spotting this.

File: common/memcached/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: memcached
-version: 0.0.1
+version: 1.0.0
```
Comment (reimannf) [inline common/memcached/Chart.yaml]: you really want to bump the version... you know everyone needs to update the requirements.yaml

Comment (auhlig) [reply common/memcached/Chart.yaml]: reverted. bad habits .. ;)

**Pull request merged**